package.json

```diff
@@ -66,7 +66,8 @@
     "url": "https://github.com/birkir/prime"
   },
   "resolutions": {
-    "typescript": "3.3.1"
+    "typescript": "3.3.1",
+    "@types/react": "16.8.12"
```

Comment:
just a note that you need to pin this or else you get this error due to both @types/react@* pulling in 16.8.2 and @types/react@16.8.12:

```
@primecms/ui: Failed to compile.
@primecms/ui: /Users/dominicwatson/Sites/github/birkir/prime/packages/prime-ui/node_modules/@types/react/index.d.ts
@primecms/ui: Type error: Duplicate identifier 'LibraryManagedAttributes'.  TS2300
@primecms/ui:     2807 |         // We can't recurse forever because `type` can't be self-referential;
@primecms/ui:     2808 |         // let's assume it's reasonable to do a single React.lazy() around a single React.memo() / vice-versa
@primecms/ui:   > 2809 |         type LibraryManagedAttributes<C, P> = C extends React.MemoExoticComponent<infer T> | React.LazyExoticComponent<infer T>
@primecms/ui:          |              ^
@primecms/ui:     2810 |             ? T extends React.MemoExoticComponent<infer U> | React.LazyExoticComponent<infer U>
@primecms/ui:     2811 |                 ? ReactManagedAttributes<U, P>
@primecms/ui:     2812 |                 : ReactManagedAttributes<T, P>
```